new-pipeline/dist.html

```diff
@@ -23,185 +23,266 @@
         <header>
             <form class="navbar-form inline pull-right">
                 <a href="https://github.com/mozilla/telemetry-dashboard/issues" class="btn btn-success" title="Report a bug in Telemetry dashboards" target="_blank">Report bug</a>
-                <a href="./evo.html" class="btn btn-default" id="switch-views" title="Switch to the evolution view">Evolution View</a>
+                <!-- <a href="./evo.html" class="btn btn-default" id="switch-views" title="Switch to the evolution view">Evolution View</a> -->
                 <a href="/" class="btn btn-default btn-primary" title="Landing page for Telemetry dashboards">Dashboards Home</a>
-                <a href="./tutorial.html#HistogramDashboard" class="btn btn-default" id="tutorial" title="Usage tutorial for Telemetry dashboards">Usage Tutorial</a>
+                <a href="./tutorial.html" class="btn btn-default" id="tutorial" title="Usage tutorial for Telemetry dashboards">Usage Tutorial</a>
                 <span class="permalink-control"></span>
             </form>
             <h1>Measurement Dashboard</h1>
         </header>
-        <div class="row">
```

Comment:
The changes from here on in dist.html are not needed.

Reply:
I made these changes for #409. I'll remove them...